css/css-backgrounds/background-clip/clip-text-multi-line-ref.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+<head>
+<title>CSS Test: background-clip: text paints multi-line text in the wrong place</title>
+<link rel="help" href="https://drafts.csswg.org/css-backgrounds-4/#valdef-background-clip-text">
```

Comment:
I think generally we don't include a `<link rel=help>` in references, so probably better to remove this.  (Having it in the test is sufficient, and sometimes references are used for multiple tests that want different `<link rel=help>`.)